src/utils/Row/src/Row.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+	<div
+		:class="$s.Row"
+		v-bind="$attrs"
+		v-on="$listeners"
+	>
+		<div
```

Comment:
Since we have a "prefix" and "suffix" on inputs, it might make sense to make the icons slots a bit more generic and named the same way.  That could open up the ability also to pass other atomic controls in like checkbox, radio, toggle, etc.

Reply:
Icon = prefix